.bumpversion.cfg

```diff
@@ -0,0 +1,9 @@
+[bumpversion]
+current_version = 1.5.0
+commit = True
+tag = True
+tag_name = v{new_version}
```

Comment:
I'd like to have signed tags, but it looks like bumpversion doesn't have that - it's also not maintained anymore from what it seems... According to https://github.com/peritus/bumpversion/issues/169 [bump2version](https://github.com/c4urself/bump2version) is a commonly accepted and maintained fork.

Could you please look into using that instead, so signed tags can be turned on? :slightly_smiling_face:

Reply:
I'll look into it tomorrow. Thanks for the suggestion :).

Reply:
`bump2version` is being used in update_version.py now and .bumpversion.cfg was updated with the new `sign_tags` toggle.